### PR TITLE
Clean up + CF overrides

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds CF overrides and performs minor codebase cleanup.

## Additions

- Adds `brand-palette.less` and `cf-theme-overrides.less`.

## Removals

- Removes extraneous whitespace in doing business with us page.
- Removes erroneous extra topdoc EOF comment from
`cf-enhancements.less`.

## Changes

- Re-generates assets.

## Review

- @sebworks 
- @jimmynotjim 
et al

## Preview

No change.

## Notes

- Brings in the CF overrides, which set the stage for completing DF-1750 and DF-1766.

### CODE REVIEW

File: static/css/cf-theme-overrides.less [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+// .expandable__padded
+@expandable__padded-bg:         @gray-10;
+@expandable__padded-bg-hover:   @gray-20;
+@expandable__padded-divider:    @gray-50;
+
+// .expandable-group
+@expandable-group_header-text:  @gray;
+@expandable-group_header-bg:    @gray-10;
+@expandable-group-bg: @white;
```
Comment (KimberlyMunoz) [inline static/css/cf-theme-overrides.less]: This line has inconsistent spacing with the rest of the document.

Comment (anselmbradford) [inline static/css/cf-theme-overrides.less]: Good catch. Fixed in https://github.com/cfpb/cfgov-refresh/commit/6d4c3ba4a051937f5ede387ea6c3f7f6670bd1fc and issue opened in https://github.com/cfpb/generator-cf/issues/62

**Comment by jimmynotjim** [comment]
This is looking good. Any idea why there are changes to main.js though? (I'm thinking grunt didn't run during a recent change).

**Comment by anselmbradford** [comment]
@jimmynotjim Yeah, I forgot to run `grunt` in https://github.com/cfpb/cfgov-refresh/pull/465

**Comment by jimmynotjim** [comment]
Ok, just making sure there wasn't something sneaking in here.

**Comment by anselmbradford** [comment]
GTG otherwise?

**Comment by jimmynotjim** [comment]
Yep, :+1: from me. @KimberlyMunoz or @sebworks ??

**Comment by anselmbradford** [comment]
One :+1: is good enough for me in this case because project standardize will revisit these files :) Squashin' in mergin'!

**Comment by KimberlyMunoz** [comment]
Looks good. One weird quirk is that I get a different `static/js/main.min.js` when I run grunt than what's in this pull request.

**Comment by anselmbradford** [comment]
Ran through https://github.com/cfpb/cfgov-refresh#updating-all-dependencies and didn't note any changes ¯\_(ツ)_/¯

**Comment by KimberlyMunoz** [comment]
:shrug: 

I just ran through all those steps and I still get a different main.min.js.

**Comment by ascott1** [comment]
Could it be a different dependency version being downloaded (since @KimberlyMunoz probably npm/bower installed more recently)?

**Comment by jimmynotjim** [comment]
Oh, we'll need to update that dependency bit when we merge `project-standardize`. The generator had a slightly different process concatenating for CF modules.

**Pull request merged**